# Statistical Data Analysis

## Project 1

Author: Zach Eckert

### Part 2

The following code is used to read the data from the text file

In [1]:
data = read.table("StatVillage.txt",sep="\t",header=TRUE)

#### 1.1 **Use the random seed that you used in Part 1 to randomly collect 20 samples of size 25 with replacement from the population of all households and compute the mean of the variable TOTINCH for each sample. List the sample means for your first 5 samples.**

In [2]:
mySeed = 967344
x = data$TOTINCH

In [3]:
vector.of.sample.means=rep(0,20)
set.seed(mySeed)
for (j in 1:20){
i=sample(1:1024,25,replace=TRUE)
this.sample=x[i]
vector.of.sample.means[j]=mean(this.sample)
}
print(vector.of.sample.means[1:5])

[1] 45398.92 59855.88 60591.28 48437.68 78246.36


The sample means for my first 5 samples are \\$45,498.92, \\$59,855.88, \\$60,591.28, \\$48,437.68, and \\$78,246.36

#### 1.2 **Compute the mean and standard deviation of your 23 sample means.**

In [4]:
mean.of.sample.means = mean(vector.of.sample.means)
sd.of.sample.means = sd(vector.of.sample.means)

print(mean.of.sample.means)
print(sd.of.sample.means)

[1] 57464.35
[1] 7979.93


The mean of my 20 sample means is \\$57,464.35, with a standard deviation of \\$7,979.93.

#### 1.3 **What is the true mean and standard deviation of the distribution of the sample mean? Compare this with your answers to (3).**

The true mean of the distribution of the sample mean is equal to the mean of the total population. The true standard deviation of the distribution of the sample mean is equal to the standard deviation of the total population divided by the square root of the sample size, or $$\sigma_x = \frac{\sigma}{\sqrt{n}}$$

In [8]:
true.mean = mean(x)
true.sd = sd(x)/sqrt(20)

print(true.mean)
print(true.sd)

[1] 56762.06
[1] 8475.501


The true mean of the distrubution of the sample mean is \\$56,762.06 with a standard deviation of \\$8,475.50.

In [9]:
mean.diff = abs(true.mean - mean.of.sample.means)
sd.diff = abs(true.sd - sd.of.sample.means)

print(mean.diff)
print(sd.diff)

[1] 702.2943
[1] 495.5708


The mean of my 20 sample means differs from the true mean by \\$702.29, and the standard deviation differs by \\$495.57

#### 1.4 **Create a histogram for the variable TOTINCH using all of the values from the population. Also, create a histogram for your vector of sample means using the same bins as you used for the population histogram. Compare the two histogram and comment on the shape of the distribution of the sample mean for the variable TOTINCH.**